src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalCost = days * 40;
+
+  if (days < 3) {
+    return totalCost;
+  }
+
+  if (days < 7) {
+    return totalCost - 20;
+  }
+
+  if (days >= 7) {
+    return totalCost - 50;
+  }
```

Comment:
```suggestion
  return totalCost - 50;
```
Если два предыдущих if не сработали, то это точно твой случай, и этот if не нужен

Reply:
Вынеси все числа в переменные, и дай им понятные имена